docker/build-project.sh

```diff
@@ -133,14 +137,15 @@ fi
 # Push buildhistory into machine-specific branch in the master buildhistory
 #
 if [ ! -z ${JOB_NAME+x} ]; then
```

Comment:
We should move the buildhistory enablement in this block and not keep it enabled in refkit-ci.inc. That'd make the pre/post scripts simpler and closer to what the build is using.